File: Milkproducing.cpp
#include <iostream>
#include <string>
#include "Milkproducing.h"

using namespace std;
int Milkproducing :: jlhMilkP =0;

Milkproducing::Milkproducing(int x,int y) : FarmAnimal(x,y){
    jlhMilkP++;
}
Milkproducing::~Milkproducing(){
    //cout<<"One of your Milkproducing animal has died"<<endl;
}
int Milkproducing:: getjlhMilkP(){
    return jlhMilkP;

}
 

File: java/src/LivingThings/Animals/MilkProducing.java
public class MilkProducing {
    private static int jlhMilkP =0;

    public MilkProducing(int x,int y){
        //super(x,y);
        MilkProducing.jlhMilkP++;
    }
    public static Integer getjlhMilkP() {
        return jlhMilkP;
    }
}